data/json/mapgen/godco/godco_eocs.json

```diff
@@ -0,0 +1,106 @@
+[
+  {
+    "type": "effect_on_condition",
+    "id": "godco_place_coop",
+    "recurrence": 100,
+    "global": true,
+    "condition": {
+      "and": [
+        { "u_has_var": "darryl_building_coop", "type": "general", "context": "trade", "value": "yes" },
```

Comment:
These variables should be stored on the NPC or globally. That way the constructions can still progress after you die and switch to an NPC follower.

Reply:
How do I store the variables globally?

Reply:
`{ "global_val": "var", "var_name": "my_var" }`. See an example [here](https://github.com/CleverRaven/Cataclysm-DDA/blob/bd17c48f95b8d7e467a8c413b97ff5299fae0524/data/json/npcs/factions.json#L167).

Reply:
Can timers also be stored globally?

Reply:
I've changed them to be stored both globally and on the NPCs.